linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml

```diff
@@ -29,13 +32,15 @@ severity: high
 
 identifiers:
     cce@rhel8: 80947-5
+    cce@sle12: 83001-8
```

Comment:
@abergmann OOC where did this CCE come from?

Reply:
NIST gave us a CCE number block and we started to assign those identifiers to the configuration settings.

Reply:
@abergmann would you be willing to add the cce number block under `shared/references/` like we have done for RHEL?

Reply:
@abergmann ^

This way more people can add CCE's to SLE rules.

Reply:
Ping @abergmann. Can you provide this? We would really like to get these SLE patches merged!